src/fideslang/validation.py

```diff
@@ -99,35 +84,43 @@ def deprecated_version_later_than_added(
     if not version_deprecated:
         return None
 
-    if version_deprecated < values.get("version_added", Version("0")):
+    version_added: Optional[str] = values.data.get("version_added")
```

Comment:
https://docs.pydantic.dev/latest/concepts/validators/#field-validators

The new way to get values from another field inside a field validator is through `ValidationInfo.data` - this is not always validated, populated, so you have to be careful when you're using it. In most cases, you might want to use a model validator.